src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -381,7 +403,13 @@ public function rsvp_process() {
 			wp_send_json_error( $response );
 		}
 
-		$products = (array) tribe_get_request_var( 'product_id' );
+		$products = [];
+
+		if ( isset( $_POST['tribe_tickets'] ) ) {
+			$products = wp_list_pluck( $_POST['tribe_tickets'], 'ticket_id' );
+		} elseif ( isset( $_POST['product_id'] ) ) {
+			$products = (array) $_POST['product_id'];
```

Comment:
* :warning: Detected usage of a non-sanitized input variable: $_POST['product_id']
* :no_entry_sign: Processing form data without nonce verification.

<sub>hash: 6b0828d381f5945b0452116f5cf3c4e6</sub>